provider/ec2/local_test.go

```diff
@@ -1031,75 +1031,6 @@ func (t *localServerSuite) TestDeriveAvailabilityZoneVolumeNoPlacement(c *gc.C)
 	c.Assert(zone, gc.Equals, "volume-zone")
 }
 
-func (t *localServerSuite) TestStartInstanceDistributionParams(c *gc.C) {
-	c.Skip("While Provisioner Parallelization on going.")
-	env := t.prepareAndBootstrap(c)
-
-	mock := mockAvailabilityZoneAllocations{
-		result: []common.AvailabilityZoneInstances{{ZoneName: "az1"}},
-	}
-	t.PatchValue(ec2.AvailabilityZoneAllocations, mock.AvailabilityZoneAllocations)
-
-	// no distribution group specified
-	testing.AssertStartInstance(c, env, t.ControllerUUID, "1")
-	c.Assert(mock.group, gc.HasLen, 0)
-
-	// distribution group specified: ensure it's passed through to AvailabilityZone.
-	expectedInstances := []instance.Id{"i-0", "i-1"}
-	params := environs.StartInstanceParams{
-		ControllerUUID: t.ControllerUUID,
-		DistributionGroup: func() ([]instance.Id, error) {
-			return expectedInstances, nil
-		},
-		StatusCallback: fakeCallback,
-	}
-	_, err := testing.StartInstanceWithParams(env, "1", params)
-	c.Assert(err, jc.ErrorIsNil)
-	c.Assert(mock.group, gc.DeepEquals, expectedInstances)
-}
-
-func (t *localServerSuite) TestStartInstanceDistributionErrors(c *gc.C) {
-	c.Skip("While Provisioner Parallelization on going.")
-	env := t.prepareAndBootstrap(c)
-
-	mock := mockAvailabilityZoneAllocations{
-		err: fmt.Errorf("AvailabilityZoneAllocations failed"),
-	}
-	t.PatchValue(ec2.AvailabilityZoneAllocations, mock.AvailabilityZoneAllocations)
-	_, _, _, err := testing.StartInstance(env, t.ControllerUUID, "1")
-	c.Assert(errors.Cause(err), gc.Equals, mock.err)
-
-	mock.err = nil
-	dgErr := fmt.Errorf("DistributionGroup failed")
-	params := environs.StartInstanceParams{
-		ControllerUUID: t.ControllerUUID,
-		DistributionGroup: func() ([]instance.Id, error) {
-			return nil, dgErr
-		},
-		StatusCallback: fakeCallback,
-	}
-	_, err = testing.StartInstanceWithParams(env, "1", params)
-	c.Assert(errors.Cause(err), gc.Equals, dgErr)
-}
-
-func (t *localServerSuite) TestStartInstanceDistribution(c *gc.C) {
```

Comment:
should this be replaced with one that tests the use of args.AvailabilityZone?

Reply:
testStartInstanceSubnet(), testStartInstanceAvailZone(), prepareAndBootstrapWithConfig() have been updated to use the args.AvailabilityZone instead of placement.